examples/service-clients/azure-client/external-controller/tests/index.ts

```diff
@@ -16,7 +16,7 @@ import {
 	LocalDeltaConnectionServer,
 } from "@fluidframework/server-local-server";
 
-import { IFluidContainer } from "@fluidframework/fluid-static";
+import type { IFluidContainer, ContainerSchema } from "@fluidframework/fluid-static";
```

Comment:
Import from `fluid-framework`?

Reply:
(And preferably remove direct dependency on `fluid-static`

Reply:
I don't know what the replacement is for the /internal imports.

Reply:
These we can correct :)